elasticdl_preprocessing/layers/normalizer.py

```diff
@@ -59,6 +59,7 @@ def call(self, inputs):
         return normalized_tensor
 
     def _normalize_fn(self, x):
-        subtractor = tf.cast(self.subtractor, x.dtype)
-        divisor = tf.cast(self.divisor, x.dtype)
+        x = tf.cast(x, tf.float64)
+        subtractor = tf.cast(self.subtractor, tf.float64)
+        divisor = tf.cast(self.divisor, tf.float64)
```

Comment:
Deep learning usually does not use float64. What is the reason to use it instead of float32?

Reply:
I have fixed it in the #1952